### PR TITLE
feature: use yarn and enable browser env linting

### PR DESCRIPTION
**Pull request by rahulreddy**

### CODE REVIEW

**Comment by rahulreddy** [comment]
@bert-e wait

**Review by alexanderchan-scality — APPROVED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -1,2 +1 @@
-{ "extends": "scality", "env": { "jest": true } }
-
+{ "extends": "scality", "env": { "jest": true, "browser": true } }
```
Comment (rachedbenmustapha) [inline .eslintrc]: apparently to ensure that it works in both node and the browser we would need the `shared-node-browser` env

Comment (rahulreddy) [reply .eslintrc]: I was just looking it up..yup.

**Review by rachedbenmustapha — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e reset

**Pull request merged**